pennylane_lightning/lightning_gpu/lightning_gpu.toml

```diff
@@ -109,3 +109,6 @@ dynamic_qubit_management = false
 # in a single execution
 non_commuting_observables = true
 
+# Whether the compilation pass should optimize the first
+# state preparation as a SetState operation.
```

Comment:
Let's not describe it as an optimization:
```suggestion
# Whether the device supports (arbitrary) initial state preparation.
```

Reply:
Changed.

Reply:
https://github.com/PennyLaneAI/pennylane-lightning/pull/826/commits/afe6d841a1977de4974d80ea143d574e24e70536